.github/workflows/testsuite.yml

```diff
@@ -373,7 +373,51 @@ jobs:
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           cd win32
-          nmake CCTYPE=MSVC142 test
+          nmake CCTYPE=MSVC142 CFG=Debug test
+
+  #===============================================
+
+  windows-msvc100:
+    name: "Windows msvc100"
+    runs-on: windows-latest
+    timeout-minutes: 120
+    needs: sanity_check
```

Comment:
Did you intend to reinstate the broken `msvc100` test entry here?

(I suspect a rebase of an old, old branch.)

Reply:
Indeed, your suspicion is correct, and I didn't pay attention.